contracts/Synthswap.sol

```diff
@@ -64,13 +71,16 @@ contract SynthSwap is ISynthSwap {
         
         // synthetix exchange
         IERC20(sUSD).approve(address(Synthetix), amountOut);
-        Synthetix.exchangeWithTrackingForInitiator(
+        uint amountReceived = Synthetix.exchangeWithTrackingForInitiator(
```

Comment:
this is a good starting point but you are not actually making the swap into sUSD yet

Reply:
and I think your TODO comment on slippage is important

Reply:
implemented slippage TODO